static/oas/v1/team/team.yml

```diff
@@ -0,0 +1,127 @@
+paths:
```

Comment:
I don't see the Open API RapiDoc working for Teams, is it working for you?

Reply:
we resolved this one. It is now showing up.